--- a/drivers/misc/mediatek/gpu/gpu_mali/mali_midgard/Makefile
+++ b/drivers/misc/mediatek/gpu/gpu_mali/mali_midgard/Makefile
@@ -14,4 +14,4 @@
 MTK_GPU_VERSION = $(subst ",,$(CONFIG_MTK_GPU_VERSION))
 $(info *MTK_GPU_VERSION 2 = $(word 2,$(MTK_GPU_VERSION)-$(word 3,$(MTK_GPU_VERSION))))
 
-obj-y += $(word 2,$(MTK_GPU_VERSION))-$(word 3,$(MTK_GPU_VERSION))/
+obj-y += $(word 2,$(MTK_GPU_VERSION))-$(word 3,$(MTK_GPU_VERSION))/drivers
