diff --git a/drivers/misc/mediatek/connectivity/common_detect/mtk_wcn_stub_alps.c b/drivers/misc/mediatek/connectivity/common_detect/mtk_wcn_stub_alps.c
index a967fbc15..c405abc6f 100644
--- a/drivers/misc/mediatek/connectivity/common_detect/mtk_wcn_stub_alps.c
+++ b/drivers/misc/mediatek/connectivity/common_detect/mtk_wcn_stub_alps.c
@@ -326,8 +326,8 @@ int mtk_wcn_cmb_stub_query_ctrl(void)
 
 	if (cmb_stub_thermal_ctrl_cb)
 		temp = (*cmb_stub_thermal_ctrl_cb) ();
-	else
-		CMB_STUB_LOG_WARN("[cmb_stub] thermal_ctrl_cb null\n");
+//	else
+//		CMB_STUB_LOG_WARN("[cmb_stub] thermal_ctrl_cb null\n");
 
 	return temp;
 }
